terraform/exoscale/modules/kubernetes-cluster/main.tf

```diff
@@ -62,7 +62,7 @@ resource "exoscale_compute" "master" {
   display_name    = "${var.prefix}-${each.key}"
   template_id     = data.exoscale_compute_template.os_image[each.key].id
   size            = each.value.size
-  disk_size       = 50
+  disk_size       = each.value.provider_settings.disk_size
```

Comment:
We could make these default to 50 if unset to make it backwards compatible without having to change the configuration, right?